material-overrides/assets/stylesheets/polkadot.css

```
@@ -1447,6 +1447,10 @@ div.mermaid {
   width: 0.9rem !important;
 }
 
+.mantine-1wkdfe1 {
```

We could add a small comment here to clarify that this belongs to kapa styles, `1wkdfe1` is a bit confusing imo. WDYT?

```suggestion
/** Kapa widget styling**/
.mantine-1wkdfe1 {
```

---

There is already a note about this on line 1436